cognite/neat/graph/loaders/_rdf2dms.py

```diff
@@ -0,0 +1,191 @@
+import json
+import warnings
+from collections import defaultdict
+from collections.abc import Iterable
+from pathlib import Path
+from typing import Any, cast
+
+import yaml
+from cognite.client import CogniteClient
+from cognite.client import data_modeling as dm
+from cognite.client.data_classes.data_modeling import ViewId
+from pydantic import HttpUrl, TypeAdapter, ValidationError, ValidationInfo, create_model, field_validator
+from pydantic.main import Model
+from rdflib.query import ResultRow
+from rdflib.term import URIRef
+
+from cognite.neat.graph.stores import NeatGraphStoreBase
+from cognite.neat.rules.issues import NeatValidationError
+from cognite.neat.rules.models import DMSRules
+from cognite.neat.rules.models.data_types import _DATA_TYPE_BY_DMS_TYPE
+
+from ._base import CDFLoader
+
+
+class DMSLoader(CDFLoader[dm.InstanceApply]):
+    def __init__(
+        self,
+        graph_store: NeatGraphStoreBase,
+        data_model: dm.DataModel[dm.View],
+        class_by_view_id: dict[ViewId, URIRef] | None = None,
+        add_class_prefix: bool = False,
+    ):
+        super().__init__(graph_store)
+        self.data_model = data_model
+        self.class_by_view_id = class_by_view_id or {}
+        self.add_class_prefix = add_class_prefix
+
+    @classmethod
+    def from_data_model_id(
+        cls,
+        client: CogniteClient,
+        data_model_id: dm.DataModelId,
+        graph_store: NeatGraphStoreBase,
+        add_class_prefix: bool = False,
```

Comment:
This requires graph to be transformed to force usage of same namespace for both properties and classes (see my comment above)